packages/mingw-w64-gcc-libs/build.sh

```diff
@@ -0,0 +1,104 @@
+TERMUX_PKG_HOMEPAGE=https://gcc.gnu.org/
+TERMUX_PKG_DESCRIPTION="Libraries coming with GCC (libgcc, libstdc++, etc.)"
+TERMUX_PKG_LICENSE="GPL-3.0, LGPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=(12.2.0)
+TERMUX_PKG_VERSION+=(2.39)  # GNU Binutils version
+TERMUX_PKG_VERSION+=(4.1.0) # GNU MPFR version
+TERMUX_PKG_VERSION+=(1.2.1) # GNU MPC version
+TERMUX_PKG_SRCURL=(https://ftp.gnu.org/gnu/gcc/gcc-${TERMUX_PKG_VERSION}/gcc-${TERMUX_PKG_VERSION}.tar.xz
+                   https://ftp.gnu.org/gnu/binutils/binutils-${TERMUX_PKG_VERSION[1]}.tar.xz
+                   https://ftp.gnu.org/gnu/mpfr/mpfr-${TERMUX_PKG_VERSION[2]}.tar.xz
+                   https://ftp.gnu.org/gnu/mpc/mpc-${TERMUX_PKG_VERSION[3]}.tar.gz)
+TERMUX_PKG_SHA256=(e549cf9cf3594a00e27b6589d4322d70e0720cdd213f39beb4181e06926230ff
+                   645c25f563b8adc0a81dbd6a41cffbf4d37083a382e02d5d3df4f65c09516d00
+                   0c98a3f1732ff6ca4ea690552079da9c597872d30e96ec28414ee23c95558a7f
+                   17503d2c395dfcf106b622dc142683c1199431d095367c6aacba6eec30340459)
+TERMUX_PKG_BUILD_DEPENDS="mingw-w64-crt"
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_NO_STATICSPLIT=true
+TERMUX_PKG_HOSTBUILD=true
+
+termux_step_post_get_source() {
```

Comment:
Hostbuild dir needs to be removed every build also, or else if won't work for repeated local builds since hostbuild step is cached